client/src/components/Alert/style.module.scss

```diff
@@ -0,0 +1,20 @@
+@use '@/styles/vars.scss' as vars;
+
+.error {
+  flex: none;
+}
+.alert {
+  display: inline flex;
+  border-radius: 2rem;
+  gap: 0.5rem;
+  align-items: flex-start;
+  justify-content: flex-start;
+  color: vars.$btn-text;
+  // Corresponds to diamond-theme/label/small
+  font-weight: 500;
+  letter-spacing: 0.5px;
+
+  color: red;
```

Comment:
should use the color from the figma. might also be a good idea to define the red error color in vars.scss